packages/text-field/Input.js

```diff
@@ -44,6 +51,10 @@ export default class Input extends React.Component {
     if (value) {
       handleValueChange(value, () => foundation.setValue(value));
     }
+    if (isValid !== undefined && foundation.setUseNativeValidation) {
```

Comment:
when would the foundation.setUseNativeValidation not exist?

Reply:
This was me just getting the sample to you. Will remove soon.

The only place this doesn't exist is during the tests when we overwrite the foundation(s) manually.